raiden/api/python.py

```diff
@@ -0,0 +1,693 @@
+# -*- coding: utf-8 -*-
+import gevent
```

Comment:
This file could be an interesting source of inspiration for an RPC protocol. It exposes all necessary API for a CLI interface.